### DISCUSSION TITLE
Result Missing from screws_tilt Display

### DISCUSSION THREAD

**Discussion by thestip** [KlipperScreen/KlipperScreen#1360] [General]
I'm running OpenNept4une, the same image everyone else uses for it, but I seem to be the only one that's missing 1 result from the screws_tilt readout; the front left. The macro works, I can see all 3 results in the web UI and the console, just not in KlipperScreen.

I've removed KlipperScreen and reinstalled it, updated everything in Kiauh, changed skins, changed screen devices... same issue. Any ideas? 

[screws_tilt_adjust]
screw1: 56.75, 182.05  
screw1_name: rear left screw
screw2: 56.75, 12.05  
screw2_name: front left screw
screw3: 226.75, 12.05  
screw3_name: front right screw
screw4: 226.75, 182.05  
screw4_name: rear right screw
horizontal_move_z: 5
speed: 150
screw_thread: CW-M4

![screws-tilt-info-missing-on-klip (1)](https://github.com/KlipperScreen/KlipperScreen/assets/45801378/d92a445c-ac15-4667-8989-2e827a28ba18)
(same on 2 different devices)

![screws-tilt-info-missing-on-klip (2)](https://github.com/KlipperScreen/KlipperScreen/assets/45801378/79f0fec9-1910-4e9c-b794-47a82b43f021)
(Fluidd)

![screws-tilt-info-missing-on-klip](https://github.com/KlipperScreen/KlipperScreen/assets/45801378/2677ce7f-4398-44e9-9d63-44f17dc8f3f2)
(console)

**Comment by alfrix** [comment]
in general when there is an issue, i need to see KlipperScreen.log

there were many changes to that panel in the last month or so, you need to be on the latest version since i incorporated many fixes that fixed this issue (apart from the support to axis inversion)

**Reply by rbc1225** [reply]
Instead of this panel 
![Screenshot 2025-02-11 174948](https://github.com/user-attachments/assets/bc904cc2-4b5f-4dd4-9998-eb359aca56ce)
I get this panel when trying to setup a simple bed leveling session like the old days. 
![476308943_1770688213780426_6838307414376323814_n](https://github.com/user-attachments/assets/e4a166c9-1df1-401e-ad41-23139a431b31)
Using the example settings just to get a feel for it. This does not move at all how I would expect. I am running an Ender 6 with a corexy configuration. Sorry, I could not find the klipperscreen log file you asked for above. 

I am running v0.4.5-42 of KlipperScreen. I think that's the latest, or that's what Kiauh says. 

Thanks
